manager-bundle/src/Resources/skeleton/config/security.yml

```diff
@@ -51,7 +51,7 @@ security:
 
     # Backwards compatibility
```

Comment:
Remove this comment then?

Reply:
dunno what that is for actually 😂 @leofeyer ?

Reply:
Can be removed IMO, introduced in here: https://github.com/contao/contao/commit/024e0938ef7e94159cc03ba67a71f261cc0e03bc

Reply:
I did add this line in the previous PR because I thought we don't need these rules anymore.